react-frontend/src/App.css

```diff
@@ -289,11 +281,29 @@ p.footnote {
 	text-align: center;
 }
 .top-padding {
-	padding-top: 1.5rem;
+	padding-top: 1rem;
 }
 .plot-container {
 	padding-left: 20px;
 }
+.chart-spinner-container {
```

Comment:
Added a spinner for when the limit has arrived but the chart data hasn't arrived yet.